sdk/pinocchio/src/instruction.rs

```diff
@@ -41,6 +44,10 @@ pub struct ProcessedSiblingInstruction {
 #[derive(Clone)]
 pub struct Account<'a> {
     // Public key of the account.
+    // JC nit: in a few other places, you're using a reference to stand in for
+    // a pointer, ie the `pubkey` in `AccountMeta` -- could we do the same thing
+    // here for all the pointers? That way we could avoid the need for PhantomData.
+    // I just don't think it'll work for data, but the others should work.
```

Comment:
Here it is using pointers since we need one for the `data`, so we don't mix the notation on the same type. We would still need the `PhantomData` for the `data` pointer.